OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -376,6 +376,11 @@ static void RemoveInfluenceInner(ref InfluenceNode influenceNode, Actor toRemove
 
 		public void Tick(Actor self)
 		{
+			UpdateTriggers();
+		}
+
+		public void UpdateTriggers()
```

Comment:
You'll need to expand the interface to the ActorMap to deal with that.  Brute-forcing the timer updates isn't a fix that we can merge.